app/controllers/reviews_controller.rb

```diff
@@ -1,11 +1,11 @@
 class ReviewsController < ApplicationController
 
   def new
-    @shelter = Shelter.find(params[:id])
+    @shelter = Shelter.find(params[:shelter_id])
```

Comment:
if I remember correctly, I believe you said that shelter_id comes into existence from the route pathing, this is a bit confusing for me